debpkg.go

```diff
@@ -22,43 +22,58 @@ import (
 // DebPkg holds data for a single debian package
 type DebPkg struct {
 	debianBinary string
-	control      debPkgControl
-	data         debPkgData
-	digest       debPkgDigest
+	control      control
+	data         data
+	digest       digest
+}
+
+var debpkgTempDir = "" // default temporary directory is os.TempDir
+
+// SetTempDir sets the temporary directory for intermediate files. When the directory doesn't
+//  exist it is automaticly created. The caller is responsible for the cleanup of tempdir.
+func SetTempDir(dir string) error {
+	if dir == "" {
```

Comment:
When the directory is emtpy you take the empty-directory as TempDir? Don't you want to get the system-wide temp-dir with `os.TempDir()`?